server/controllers/finance/reports/balance/index.js

```diff
@@ -154,6 +154,7 @@ async function getBalanceForFiscalYear(period, currencyId) {
       WHERE pt.fiscal_year_id = ? AND p.fiscal_year_id = ?
       GROUP BY pt.account_id
     )s ON a.id = s.account_id
+    WHERE a.locked <> 1 OR a.hidden <> 1
```

Comment:
👍

Reply:
Do we have rules that prevent someone from locking or hiding an account with a balance?  If not, we need to implement them with this change.

The risk is that someone will hide an account with a balance and we'll never know why their balance sheet was incorrectly balanced.